config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: REPLACE_IMAGE
+  newTag: TAG
```

Comment:
I don't like that we commit these placeholders. But I'm not sure how to avoid it.

Reply:
I am not sure if this file is necessary. Operator sdk generates lots of files. These placeholders are needed because they are used and replaced by HCO

Reply:
The file is necessary, it contains the image and tag that will be used in Deployment manifest.
I don't like that there are multiple makefile targets  that write different data here.
`make olm-catalog` writes these placeholders, but `make deploy` and `make bundle` write the `${IMG}` variable.

But for it is ok for now. Maybe we can solve it later.